source/php/Controller/Project.php

```diff
@@ -42,6 +42,17 @@ public function singleViewController($data)
         );
 
         $data['scrollSpyMenuItems'] = $this->scrollSpyMenuItems($data);
+        $data['list'] = [];
```

Comment:
`$data['list']` should be scoped within `$data['project']` eg. `$data['project']['gallery']`

Suggestion/possible solution: 
1. Ensure we get either the gallery postMeta or an empty array
`$data['project']['gallery'] = WP::getPostMeta('gallery', [])` 
2. Include UI class with use (in controller):
`use ProjectManagerIntegration\UI\Gallery`
3. Refactor controller data using a postMetaData to viewData adapter in controller:
`$data['project']['gallery'] = Gallery::create(WP::getPostMeta('gallery', []))`
4. Add new class: ProjectManagerIntegration\UI\Gallery, with a static create method that always return an array (empty or with list)  `@includeIf(!empty($data['project']['gallery']), 'partial')` 
5. We can now easily check the view for `@includeIf(!empty($data['project']['gallery']), 'partial')` 
6. Finally replace `$list` with namespaced property `$project['gallery]/$project['gallery]['list']`